MVC/system/classes/database.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+  class database
+  {
+    public $user = USER;
```

Comment:
It is not necessary to store the database information in variable since it is already defined previously. You can directly access it

Reply:
Yes but for clarity purpose I used.